notification.py

```diff
@@ -281,8 +282,43 @@ def notify(self):
                 config.TWITTER_ACCESS_KEY and
                 config.TWITTER_ACCESS_SECRET):
             tweeted = self.tweet()
+        if config.TELEGRAM_BOT_TOKEN and config.TELEGRAM_CHAT_ID:
+            telegram = self.sendToTelegram()
 
-        return tweeted or pushed
+        return tweeted or pushed or telegram
+
+    def sendToTelegram(self):
+        try:
+            TELEGRAM_BASE_URL = "https://api.telegram.org/bot{token}/{command}"
+            title = self.name
+            if self.expire_time:
+                minutes, seconds = divmod(self.delta.total_seconds(), 60)
+                description = 'Expire at: {e} ({m}m{s:.0f}s left)'.format(e=self.expire_time, m=int(minutes), s=seconds)
+            else:
+                description = ("It'll expire between {e1} & {e2}.").format(e1=self.min_expire_time, e2=self.max_expire_time)
+            
+            if self.iv[0] is not None:
+                title += ' ({a}/{d}/{s})'.format(a=self.iv[0], d=self.iv[1], s=self.iv[2])
```

Comment:
```python
title += ' ({0[0]}/{0[1]}/{0[2]})'.format(self.iv)
```
Would be more concise and would [keep the line under 79 characters](https://www.python.org/dev/peps/pep-0008/#maximum-line-length), which I usually try to do if it's not too much hassle.